### PR TITLE
Feature/speach to text

### PR DESCRIPTION
**Pull request by mikiisz**

### CODE REVIEW

**Comment by mikiisz** [comment]
This is a basic infra for our speech to text transcript.
TODO: 
* add python logic to use AWS Transcript API
* record voice and send to gateway via POST

File: app.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
     "slug": "doctorMyEyes",
     "version": "1.0.0",
     "orientation": "portrait",
-    "icon": "./assets/icon.png",
+    "icon": "./src/assets/icon.png",
```
Comment (moniad) [inline app.json:7]: should we move assets to src? because these files were originally generated here

File: config.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "TRANSCRIPT_GATEWAY": "replace with api url, keep it away from git history"
```
Comment (mikiisz) [inline config.json:2]: @moniad dam ci url na privie

File: .gitignore [hunk 3]
```diff
@@ -1,12 +1,14 @@
 node_modules/
-.expo/
+.expo
+.expo-shared
```
[the remark this answers is not in the document]
Comment (mikiisz) [reply .gitignore:3]: reger that

[the remark this answers is not in the document]
Comment (mikiisz) [reply .gitignore:3]: roger that

File: app.json (hunk 1 above)
Comment (mikiisz) [reply app.json:7]: i will move it

Comment (mikiisz) [reply app.json:7]: assets should be in src section, they are part of nodeJS, the root folder is used as a wrapper between services

**Pull request merged**